digital_image_processing/image_watersheding_custom_markers.py

```diff
@@ -0,0 +1,89 @@
+import cv2
+import numpy as np
+from matplotlib import cm
+
+# loading the image (kept in the same directory)
+img = cv2.imread('/image_data/scenery.jpg')
+img_copy = img.copy()
+
+# defining marker image & segmented image
+marker_image = np.zeros(img.shape[:2], dtype=np.int32)
+segments = np.zeros(img.shape, dtype=np.uint8)
+
+"""
+creating function for defining color
+"""
+
+
+def create_rgb(i):
+    return np.array(cm.tab10(i)[:3]) * 255
+
+
+# create color tuple for each  key 0-9
+colors = []
+for i in range(10):
+    colors.append(create_rgb(i))
```

Comment:
```suggestion
colors = [create_rgb(i) for i in range(10)]
```
A list comprehension.